src/refract/RenderJSONVisitor.cc

```diff
@@ -8,6 +8,7 @@
 
 #include "Element.h"
 #include "Visitors.h"
+#include "../Serialize.h"
```

Comment:
I included that so I could do something like `drafter::SerializeKey::Nullable` instead of `"nullable"`, however based on your other comments I'll just switch to the plain string.